.github/ISSUE_TEMPLATE/pre-release-demos-nightly.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Pre-Release Demo Testing on Nightly
+about: This template can be used to track the testing of demos on nightly leading up to the next Stackable release
+title: "chore(tracking): Test demos on nightly versions"
+labels: ['epic']
+assignees: ''
+---
+
+<!--
+    Make sure to update the link in '.github/ISSUE_TEMPLATE/release.md' when
+    you change the front matter above.
+-->
+
+<!--
+    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
+    This was created by an issue template: https://github.com/stackabletech/issues/issues/new/choose.
+-->
+
+## Pre-Release Demo Testing on Nightly
+
+Part of <https://github.com/stackabletech/issues/issues/TRACKING_ISSUE>
+
+For each demo, run the following commands:
+
+```shell
+# Install demo
+stackablectl demo <DEMO_NAME>
+
+# Uninstall operators
+stackablectl release uninstall <CURRENT_RELEASE>
+
+# Update CRDs to nightly version (on main)
+# Repeat this for every operator used by the demo
+kubectl replace -f https://raw.githubusercontent.com/stackabletech/commons-operator/main/deploy/helm/commons-operator/crds/crds.yaml
```

Comment:
Maybe add in a placeholder to prevent auto-brain from only doing one thing.

```suggestion
kubectl replace -f https://raw.githubusercontent.com/stackabletech/commons-operator/main/deploy/helm/commons-operator/crds/crds.yaml
kubectl replace -f https://raw.githubusercontent.com/stackabletech/...-operator/main/deploy/helm/...-operator/crds/crds.yaml
```